turbinia/api/models/request_status.py

```diff
@@ -63,17 +64,28 @@ def get_request_data(
         if current_request_id == request_id:
           self.tasks.append(task)
 
+    if len(tasks) >= 1 and tasks[0].get('all_args'):
```

Comment:
What happens here if tasks is empty, or if 'all_args' is not present?
Maybe add a comment explaining why we want to grab the first task and then you compare start_ttime a few lines below, just so it's clear to someone else reading the codebase.